data/human/transport missions.txt

```diff
@@ -3300,3 +3300,42 @@ mission "Spacediving amateurs"
 	on complete
 		payment 30000
 		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."
+
+
+
+mission "Courier 5"
+	name "Yottrite samples to <planet>"
+	deadline
+	description "Deliver a few Yottrite samples to <destination> by <date>. Payment is <payment>."
+	minor
+	cargo "Yottrite samples" 10
+	to offer
+		random < 5
+	source
+		government "Quarg"
+	destination
+		distance 2 13
+		government "Republic"
+		attributes "rich"
+	on offer
+		conversation
+			`A man waves at you from afar, trying to get your attention."`
+			choice
+				`	(Find out what he wants.)`
+				`	(Ignore him.)`
+					decline
+			`	"My name is Jason ," he says. "I've gotten my hands on a few Yottrite samples that I bought from a Quarg. We need you to send them to a research center."`
+			choice
+				`	"I'd be glad to."`
+				`	"Sorry, I'm not interested in that sort of work."`
+					decline
+			`	"Excellent!" he says. "Come, let my men load it up onto your ship. An associate of mine will pick them up at <destination>." He adds, "They are somewhat unstable, so be careful with them or they could explode."`
```

Comment:
Perhaps expand this mission by allowing the player to ask what the samples are for, and what yottrite is.

Reply:
Indeed.

Reply:
ok i gonna work on it